gke/enterprise/mesh/main.tf

```diff
@@ -0,0 +1,39 @@
+/**
+* Copyright 2025 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_enterprise_mesh]
+resource "google_project_service" "default" {
+  for_each = toset([
+    "meshconfig.googleapis.com"
+  ])
+
+  service            = each.value
+  disable_on_destroy = false
+}
+
+resource "google_gke_hub_feature" "default" {
+  name     = "servicemesh"
+  location = "global"
+
+  fleet_default_member_config {
+    mesh {
+      management = "MANAGEMENT_AUTOMATIC"
+    }
+  }
+
+  depends_on = [google_project_service.default]
+}
+# [END gke_enterprise_mesh]
```

Comment:
google_gke_hub_feature_membership is not needed for 'enterprise' ?

Reply:
Hi @yliaog - This example is setting a fleet default (only part of Enterprise), perhaps I should rename the snip tag for clarity?

Reply:
Maybe `gke_enterprise_mesh_default`?

Reply:
ok, I see, it is for fleet default. I think it's fine to leave as is.